src/main/java/Server.java

```diff
@@ -36,4 +34,9 @@ public void start() {
     }
   }
 
+  public void close() throws IOException {
```

Comment:
I added this method while I was having trouble with Cucumber tests running and completing once but not every other time. I created this method thinking it would help solve that issue. Since my Cucumber tests are no longer starting up the server, the method is not necessary, so I've deleted it now.